### DISCUSSION TITLE
Wrong static IP

### DISCUSSION THREAD

**Discussion by Adi-18** [telekom-security/tpotce#1327] [Q&A]
I installed T-Pot from scratch with ISO-File tpot_amd64.iso (22.04).

Its behind of a DMZ with static ip for NAT.
My IP for T-Pot must to be **192.168.72.254**, Default route 192.168.72.1.
Setup was successfully, bit now there is no access to Dashboard because of wrong ip address: **172.17.0.1**

**Comment by Adi-18** [comment]
This is the problem:
![image](https://github.com/telekom-security/tpotce/assets/29123730/7a7a18fc-056e-4b95-b72e-2ef9c1b2c1b8)

What is the solution?

**Comment by Epicism** [comment]
I'm experiencing the same issue. My /etc/network/interfaces had an entry to a non-existent ens18 interface. I commented it out,  un-commented the reference to eth0, and changed the values to match my installation.

![image](https://github.com/telekom-security/tpotce/assets/10110013/28efb2ea-a38a-4e17-9b89-d3e305db8ffa)

Edit: Nevermind, the changes did not stick...

**Reply by t3chn0m4g3** [reply]
#1317
